### ISSUE TITLE
Add possibility to create scale-out magnetic "cloudsigma_drive"

### ISSUE THREAD

**Issue by wojcik92michal** [cloudsigma/terraform-provider-cloudsigma#28]
# Feature request

### Description

According to the docs (version `1.6.0`, https://registry.terraform.io/providers/cloudsigma/cloudsigma/latest/docs/resources/drive), seems to me that it is not possible to define a "scale-out Magnetic drive".

### New or Affected Resource(s)

* cloudsigma_drive

### Potential Terraform Configuration

```hcl
resource "cloudsigma_drive" "foobar" {
  media = "disk"
  name  = "foobar"
  storage_type = "XXX"
  size  = 5 * 1024 * 1024 * 1024 # 5GB
}
```

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)
  to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by pavel-github** [comment]
@wojcik92michal, good catch.

At the moment `storage_type` parameter is computed only and will be used by *read operation. We should consider it by *create operation as well.

**Comment by mjtrangoni** [comment]
@pavel-github this should be solved merging #51.

**Comment by pavel-github** [comment]
@wojcik92michal, fyi, it's available from version **1.11.0**.
I'm going to close the issue. Thank you for your input.
